github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -488,7 +488,12 @@ presubmits:
       preset-docker-mirror-proxy: "true"
       preset-gcs-credentials: "true"
       preset-pgp-bot-key: "true"
-    cluster: ibm-prow-jobs
+      preset-podman-shared-images: "true"
+    cluster: kubevirt-prow-workloads
+    extra_refs:
+    - org: kubevirt
+      repo: kubevirt
```

Comment:
Using kubevirt here allows us to use the shared local kubevirtci images rather than having to download the image for each run which reduces a lot of time.

Reply:
Ah, that makes sense. We could use the KUBEVIRTCI_TAG variable though to pin the version.